stm32_part_table.yaml

```
@@ -591,4 +591,13 @@ stm32mp1:
     rm_title: STM32MP157
     rm_url: https://www.st.com/resource/en/reference_manual/dm00366355-stm32mp157-advanced-armbased-32bit-mpus-stmicroelectronics.pdf
     members:
-      - STM32MP157
\ No newline at end of file
+      - STM32MP157
+
+stm32wl:
+  stm32wlex:
```

These have to match the module name inside the crate, so either we could rename the YAML file to stm32wlex.yaml or rename this line to stm32wle5. Given we don't know what ST will release next, probably stm32wle5 is safest.

```suggestion
  stm32wle5:
```

---

For now we can change it to `stm32wle5` like you suggested. In the future we might see a STM32WLE**6** or something, maybe then it should be stm32wle**x** again? I believe they are planning to release a dual core version with M0 and M4 cores, because they accidentally sent me SVD files for that part which is called `STM32WL5x`, so a 5 instead of the E...